test/e2e/tests/onboarding.spec.js

```diff
@@ -162,11 +163,12 @@ describe('MetaMask onboarding', function () {
         );
 
         // Check that the error message is displayed for the password fields
-        const passwordErrorIsDisplayed = await driver.isElementPresent({
-          text: "Passwords don't match",
-          css: 'h6',
-        });
-        assert.equal(passwordErrorIsDisplayed, true);
+        await driver.isElementPresent(
+          // eslint-disable-next-line prettier/prettier
```

Comment:
Nit: We can remove this comment

Reply:
oh don't know where did it come from. but removed 🙏🏻